### DISCUSSION TITLE
How do I install third party plugins when used globally?

### DISCUSSION THREAD

**Discussion by olivernybroe** [semantic-release/semantic-release#2500] [Configuration] [answered]
As I don't have a npm project, I am using the global installation and running via `npx semantic-release`.

However I am using a third-party plugin, but when using the global installation, I can't figure out how to let it pick up the global installed package. 
I installed the plugin via `npm install -g semantic-release-export-data`

**Comment by travi** [comment, marked as answer]
as long as semantic-release and plugins are all installed globally for the version of node being used at execution time, the plugins should be available to semantic-release when executing with npx. 

are you configuring semantic-release to use the plugin in the `.releaserc.json` or similar in your project?

**Reply by olivernybroe** [reply]
Seems like npx creates a unique folder for it's global installation when ran directly. 

My solution was to just do 
```
npm --location=global install semantic-release semantic-release-export-data
```

and then it works 👍
